src/mmw/apps/modeling/geoprocessing.py

```diff
@@ -71,6 +72,16 @@ def sjs_submit(host, port, args, data, retry=None):
             except MaxRetriesExceededError:
                 raise Exception('No slots available in Spark JobServer.\n'
                                 'Details = {}'.format(response.text))
+        elif error['result'] == 'context geoprocessing not found':
+            geop_context_url = '{}/contexts?reset=reboot'.format(base_url)
+            context_response = requests.put(geop_context_url)
+
+            if context_response.ok:
+                retry()
+            else:
+                raise Exception('Unable to create missing geoprocessing\n'
```

Comment:
Consider leaving out the `\n` in between the message, so the message is the first line and the details the second. The output should look like:

```
Unable to create missing geoprocessing context in Spark JobServer.
Details = [...]
```